common/src/main/java/com/wynntils/core/persisted/config/ConfigManager.java

```diff
@@ -333,4 +333,10 @@ private <T> void setConfigValue(Config<T> config, Object value) {
             config.setValue(typedValue);
         }
     }
+
+    public void userSwitched() {
```

Comment:
I think a better name would be like `reloadAllConfigs` or something like that. The method should describe what it does, not when it is called.